src/wasm/wasm.cpp

```diff
@@ -900,6 +900,47 @@ void Throw::finalize() { type = Type::unreachable; }
 
 void Rethrow::finalize() { type = Type::unreachable; }
 
+void ThrowRef::finalize() { type = Type::unreachable; }
+
+bool TryTable::hasCatchAll() const {
+  return std::any_of(
+    catchRefs.begin(), catchRefs.end(), [](bool v) { return v; });
```

Comment:
Wait, isn't an empty (not a full) `catchTag` (not `catchRef`) what indicates a catch_all? That's what I understood from `wasm.h` at least - did I get it wrong?

Reply:
You're right, good catch... Don't know what I was thinking.